### PR TITLE
[DO NOT MERGE]Update Azure cloud docs

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs-2.0/nebula-cloud/nebula-cloud-on-azure/2.how-to-create-subsciption.md [hunk 1]
```diff
@@ -4,28 +4,48 @@
...
-      |:---|:---|
-      |订阅|选择用户的订阅。|
-      |资源组|选择用户已有的资源组或新建一个资源组。|
+  | 计划 | 说明 |
+  | - | - |
+  | 30-day free trial | 提供 30 天（Solution 实际运行时间）免费试用版服务。选择该计划后，在[配置 Nebula Graph Solution](3.how-to-set-solution.md) 时可以选择 **Free** 版，不能选择其它版本。 |
+  | Beta | 付费版服务。选择该计划后，在配置 Nebula Graph Solution 时可以选择 **Enterprise** 版和 **Developer** 版，不能选择 **Free** 版。 |
 
-    - SaaS 详情信息
+  ！！！ caution
```
Comment (foesa-yang) [inline docs-2.0/nebula-cloud/nebula-cloud-on-azure/2.how-to-create-subsciption.md:18]: 好像是中文的符号

File: docs-2.0/nebula-cloud/nebula-cloud-on-azure/3.how-to-set-solution.md [hunk 2]
```diff
@@ -4,24 +4,47 @@ Solution 指运行在 Nebula Graph Cloud 上的 Nebula Graph 数据库。在 Azu
...
-  1. 在 **供应商** 选择**区域**。
+1. 完成[订阅创建](2.how-to-create-subsciption.md)，并在**订阅进度**页单击**立即配置账户**后，在 **Permissions Requested** 页单击 **Accept** 为 Nebula Graph 授予权限。
+
+2. 在**请选择 Cloud 订阅类型**页，选择需要的 Cloud 服务版本，并单击版本下方的 **Get Started Free** 或 **Create**。
+
+  可选版本如下：
+
+  | 版本 | 说明 |
+  | - | - |
+  | Free | 为在云上学习与探索 Nebula Graph 设置的 30 天免费试用版。服务为单机架构，规格固定为 DT.C2.M4（2Core），4GiB RAM，Standard SSD 50 GiB。要选择 **Free** 版，创建订阅时必须选择了 **30-day free trial** 计划。|
```
[the remark this answers is not in the document]
Comment (randomJoe211) [reply docs-2.0/nebula-cloud/nebula-cloud-on-azure/3.how-to-set-solution.md:15]: 改了

**Review by abby-cyber — APPROVED** [review]

File: docs-2.0/nebula-cloud/nebula-cloud-on-azure/2.how-to-create-subsciption.md [hunk 3]
```diff
@@ -4,28 +4,48 @@
...
+  | Beta | 付费版服务。选择该计划后，在配置 Nebula Graph Solution 时可以选择 **Enterprise** 版和 **Developer** 版，不能选择 **Free** 版。 |
 
-    - SaaS 详情信息
+  !!! caution
 
-      |设置项|预设值|
-      |:---|:---|
-      |名称|输入 SaaS 订阅创建名称。|
-      |定时收费|选择**开**或**关**。|
+      - 每个 Azure 账号只能订阅一次 **30-day free trial**，到期后无法再次订阅。
```
Comment (hetao92) [inline docs-2.0/nebula-cloud/nebula-cloud-on-azure/2.how-to-create-subsciption.md:20]: need to fix

File: mkdocs.yml [hunk 4]
```diff
@@ -214,7 +214,7 @@ extra:
     branch: release-3.1
     tag: v3.1.2
   cloud:
-    azureRelease: 2.6.2 # Azure云兼容的最新内核版本
+    azureRelease: 3.1.0 # Azure云兼容的最新内核版本
```
Comment (hetao92) [inline mkdocs.yml:217]: 3.1.1

**Comment by whitewum** [comment]
old dated

**Pull request closed without merging**